src/access/examples/ExampleAccessControlDelegate.cpp

```diff
@@ -1069,12 +1069,13 @@ class AccessControlDelegate : public AccessControl::Delegate
         CHIP_ERROR err = LoadFromFlash();
         if (err != CHIP_NO_ERROR)
         {
+            ChipLogDetail(DataManagement, "Unable to load stored ACL entries; using empty list instead");
```

Comment:
I've been switching to this kind of message for logs.
```suggestion
            ChipLogDetail(DataManagement, "AccessControl: unable to load stored ACL entries; using empty list instead");
```